### ISSUE TITLE
[Docs] Add docs for `mesheryctl system check` command

### ISSUE THREAD

**Issue by navendu-pottekkat** [meshery/meshery#4197]
#### Current State
`mesheryctl system check` is a command use to run system health checks for both pre and post deployment scenarios.

#### Desired State
1. Add documentation for this command in the [Command Reference](https://docs.meshery.io/reference/mesheryctl) page and add individual page for this command.
2. Create a [guide](https://docs.meshery.io/guides) which walks through how a user can verify their environment readiness covering most common use-cases.
---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by ishitakeshawani** [comment]
Can I  work on this issue? @navendu-pottekkat

**Comment by navendu-pottekkat** [comment]
Yes @ishitakeshawani 

@piyushsingariya Could you share any resources on what tests are run for preflight as well as post-deployment checks?

**Comment by piyushsingariya** [comment]
```
Pre-deployment
1. docker healthchecks
2. kubernetes healthchecks
3. Kubernetes Version healthchecks
Post-deployment(also run the first 3 checks mentioned)
4. Meshery Version healthchecks
5. Meshery Adapter healthcheks
```

This guide should address possible reasons behind the failure of each of these checks and their possible solutions.
You can figure out the failing reasons by reading the code, some you must have faced while setting up meshery in your system, you can ask other folks to share the issues they faced while working with the respected component of which you're writing the failure reasons, rest I'll address when the PR will be up.

**Comment by navendu-pottekkat** [comment]
@ishitakeshawani You can check this above comment as well as test out the command before you write the docs.

**Comment by ishitakeshawani** [comment]
> @ishitakeshawani You can check this above comment as well as test out the command before you write the docs.

ok. I will do it.

**Comment by alphaX86** [comment]
> @ishitakeshawani Are you working on this?
> 
> If not, @alphaX86 could you take this up quickly?

Let's wait for her reply within today. If she didn't work on this issue yet, I'll tie up loose ends today

**Comment by ishitakeshawani** [comment]
> @ishitakeshawani Are you working on this?
> 
> If not, @alphaX86 could you take this up quickly?

No, I am not working. I stuck with some work. Sorry, I forgot to mention this here.

**Comment by alphaX86** [comment]
Thanks for the update @ishitakeshawani

**Comment by alphaX86** [comment]
I'll tie up the issue
